README.md

```diff
@@ -1,50 +1,248 @@
- # REISE.jl
+# REISE.jl
 Renewable Energy Integration Simulation Engine.
 
 This repository contains, in the **src** folder, the Julia scripts to run the power-flow study in the U.S. electric grid. The simulation engine relies on [Gurobi] as the optimization solver.
 
+## Table of Contents
+1. [Dependencies](#dependencies)
+2. [Installation (Local)](#installation-local)
+3. [Usage (Julia)](#usage-julia)
+4. [Usage (Python)](#usage-python)
+5. [Docker](#docker)
+6. [Package Structure](#package-structure)
+7. [Formulation](#formulation)
+
+
 ## Dependencies
-This package requires installations of the following
-- [Julia]
-- [Gurobi]
-- [Python]
+This package requires installations of the following, with recommended versions listed.
+- [Julia], version 1.5
+- [Gurobi], version 9.1
+- [Python], version 3.8
+
+This package can also be run using Docker, in which the following are required:
+- [Docker]
+- [Gurobi Cloud License] file
 
 For sample data to use with the simulation, please visit [Zenodo].
 
+### System Requirements
+
+Large simulations (e.g. covering the entire USA grid instead of specific interconnects or with large time intervals) can constrained by the amount of RAM. As a general estimate, 1-2 GB of RAM is needed per hour in the interval in a simulation across the entire USA grid. For example, a 24-hour interval would require 24-48 GB of RAM; if only 16 GB of RAM is available, consider using a time interval of 8 hours or less. The memory necessary would also be proportional to the size of grid used, so as the Western interconnect is ~8 times smaller than the entire USA grid, it would also take 8x less RAM.
+
```

Comment:
There are a couple of sentences above that sound weird but I am not a native speaker so I am likely wrong:
* can constrained by the amount of RAM --> can require a significant amount of RAM
* As a general estimate, 1-2 GB of RAM is needed per hour in the interval --> As a general estimate, 1-2 GB of RAM is needed to solve the DCOPF problem in the full U.S. grid over 1-hour.